### DISCUSSION TITLE
yt transcript not working

### DISCUSSION THREAD

**Discussion by BLCXD** [danielmiessler/Fabric#1156] [Q&A]
Hi, i could not spot the problem. Can sb help? 

the terminal

```
C:\Users\blc>fabric yt --transcript https://youtu.be/UbDyjIIGaxQ?si=XtGudCZPkQtcsmX8 | fabric --model gemini:lastest -sp extract_wisdom
 Requested Model = gemini:lastest
 Default Model = gemini-1.0-pro-vision-latest
 Default Vendor = Gemini.

could not find vendor.

C:\Users\blc>
```

what is this **vendor**

**Comment by NuCl34R** [comment]
Make sure when you type 'fabric -L' that the model you are calling in your command is accurate. You wrote 'gemini:lastest' did you mean 'gemini:latest'. Appending the model isn't always required, if you don't specify it will use the default one, make sure it is correctly set using 'fabric --setup'.

Hope this helps pinpoint the problem, also this command: 'fabric -y https://youtu.be/UbDyjIIGaxQ?si=XtGudCZPkQtcsmX8 -p extract_wisdom' will do the same as yours, once your default AI model is set.
